php/symfony/composer.json

```diff
@@ -1,9 +1,9 @@
 {
   "require": {
-    "symfony/framework-bundle": "~5.1.0",
+    "symfony/framework-bundle": "^5.2",
```

Comment:
This could install 5.3
@see https://getcomposer.org/doc/articles/versions.md#caret-version-range-
and we must use the same version in `config.yaml`

I prefer using `~5.2.0` (not sure this is the standard in php word, btw)

Reply:
That is correct. I thought this was a feature =)

I'll revert.